avocado/utils/cloudinit.py

```diff
@@ -122,8 +130,14 @@ def iso(output_path, instance_id, username=None, password=None,
 
 
 class PhoneHomeServerHandler(BaseHTTPRequestHandler):
+    """Handles HTTP requests to the phone home server.
+    """
```

Comment:
When using one-line docstrings this should follow the format:

```
""" msg here. """
```
From PEP-0257:

> The closing quotes are on the same line as the opening quotes.

Reply:
@beraldoleal good catch. Notice that I am using your suggestion from the other PR :)